README.md

```diff
@@ -50,20 +50,41 @@ You must pass a `src` property with the link of the image:
 
 That image will be loaded as soon as the image enters the viewport.
 
-## Progressive loading
+## Responsive Images
+
+Using the `srcset` prop in combinations with the `sizes` attribute you can set images for different sizes:
+
+```html
+<template>
+  <v-lazy-image
+    srcset="image.jpg 1x, image_2x.jpg 2x"
+    ></v-lazy-image>
+</template>
+```
+
+It's possible to use it with the `sizes` attributes as well:
+
+```html
+<template>
+  <v-lazy-image
+    srcset="image.jpg 400w, image.jpg 800w"
+    sizes="(max-width: 400w) 400w, 100vw"
+    ></v-lazy-image>
+</template>
+```
+
+## Progressive Loading
```

Comment:
Maybe is good to place the *Progressive Loading* before the *Responsive Images* and then in the **